packages/components/src/theme/ComposerTheme/Components/Input.ts

```diff
@@ -111,6 +111,11 @@ export const inputGroupStyles = {
 };
 
 const calculateSize = (size: string) => {
+  const inputSizeHeight = {
+    small: '2rem',
+    medium: '2.5rem',
+    large: '3rem',
+  };
```

Comment:
can we make this match the input definition instead of hardcoding this values?

Reply:
I've double checked how `chakra-ui` deals with this situation and they use an `--input-height` token to specify the `height` of the `Input`. We are using the same methodology to highlight an input as chakra by using borders, outlines or boxShadows depending on the use case.

So I'd say we need to specify the `height` as they do.  I'm not sure if we want to add a new token or keep the hardcoded value in the theme.